qiskit_experiments/library/characterization/t2ramsey.py

```diff
@@ -144,7 +126,7 @@ def circuits(self) -> List[QuantumCircuit]:
             circ.metadata = {
                 "experiment_type": self._type,
```

Comment:
Can we cleanup metadata with this PR?

Reply:
Let's do a follow up since I think this PR is just about ready :slightly_smiling_face:.